cw_decred/lib/wallet.dart

```diff
@@ -487,12 +526,13 @@ abstract class DecredWalletBase
     }
     var addr = address;
     if (addr == null) {
-      addr = libdcrwallet.currentReceiveAddress(walletInfo.name);
+      addr = walletAddresses.address;
     }
     if (addr == null) {
```

Comment:
`walletAddresses.address` returns an empty string, not null, currently.

So the app shows `Exception: unable to decode address: address "" is not a supported type` currently.

Reply:
Thanks fixed.